pkgs/by-name/wi/wideriver/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  gcc,
+  wayland,
+  wayland-scanner,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wideriver";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "alex-courtis";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-Jn3DfflDlMSZMlSRBfzQeEUKgYLctd/PiQRWH8uwtlU=";
+  };
+
+  buildInputs = [
+    gcc
```

Comment:
```suggestion

```
This doesn't need to be included, it's already included in `stdenv`.

Reply:
Oops, sorry, I wrote that confusingly. Please undo the `buildInputs` -> `nativeBuildInputs` change.